README.md

```diff
@@ -1,65 +1,89 @@
 <div align="center">
     <div>
-        <!-- Load images from raw.githubusercontent.com to enable image rendering when viewed from https://www.npmjs.com/package/@ni/eslint-config -->
-        <img src="https://raw.githubusercontent.com/ni/javascript-styleguide/HEAD/resources/logo.svg" alt="JavaScript, TypeScript, and NI logo" width="300" height="100">
+        <!-- Load images from raw.githubusercontent.com to enable image rendering when viewed from npmjs.com -->
+        <img src="https://raw.githubusercontent.com/ni/javascript-styleguide/HEAD/docs/logo.svg" alt="JavaScript, TypeScript, and NI logo" width="300" height="100">
     </div>
 </div>
 
 # NI JavaScript and TypeScript Style Guide
 
-[![NPM Version](https://img.shields.io/npm/v/@ni/eslint-config.svg)](https://www.npmjs.com/package/@ni/eslint-config)
+[![NPM Version](https://img.shields.io/npm/v/@ni/eslint-config-javascript.svg)](https://www.npmjs.com/package/@ni/eslint-config-javascript)
+[![NPM Version](https://img.shields.io/npm/v/@ni/eslint-config-typescript.svg)](https://www.npmjs.com/package/@ni/eslint-config-typescript)
+[![NPM Version](https://img.shields.io/npm/v/@ni/eslint-config-angular.svg)](https://www.npmjs.com/package/@ni/eslint-config-angular)
 
 Welcome to NI's JavaScript and TypeScript linter rules for [ESLint](https://eslint.org/docs/user-guide/getting-started).
 
 ## Installation
 
-Install `@ni/eslint-config` and its peer dependencies.
+If you are using npm 7 or greater, install the package for your corresponding language:
 
-Use [`npm view`](https://docs.npmjs.com/cli/view.html) to list the correct versions of each package to install yourself.
+- For JavaScript run: 
 
-```bash
-npm view @ni/eslint-config peerDependencies
-```
+    ```bash
+    npm install @ni/eslint-config-javascript
+    ```
 
-Alternatively, use [`npx install-peerdeps`](https://www.npmjs.com/package/install-peerdeps) as a shortcut to install the packages for you.
+   Then follow the [JavaScript configuration](#javascript-configuration) instructions.
+- For TypeScript run:
+
+    ```bash
+    npm install @ni/eslint-config-typescript
+    ```
+
+    Then follow the [TypeScript configuration](#typescript-configuration) instructions.
+- For Angular run:
+
+    ```bash
+    npm install @ni/eslint-config-angular
+    ```
+
+    Then follow the [Angular configuration](#angular-configuration) instructions.
+
+If you are using npm 6 or lower, use the following instructions to manually install peer dependencies:
```

Comment:
We could be a bit more explicit that the npm 6 workflow also includes the steps above to pick the right package and to follow the language-specific configuration instructions. Right now it reads like those steps are only needed for Angular 7. Maybe reframe the npm 6 steps as an additional step in between running the install command and following the configuration instructions?

Reply:
Clarified that all projects need to install a lint package but the manual peer dependencies install is an additional step specific to npm 6 and below